inject-groovy/src/main/groovy/io/micronaut/ast/groovy/visitor/GroovyVisitorContext.java

```diff
@@ -309,4 +310,14 @@ private SyntaxErrorMessage buildErrorMessage(String message, Element element) {
         return attributes.get(name, conversionContext);
     }
 
+    @Override
+    public Collection<String> getGeneratedResources() {
+        return Collections.unmodifiableCollection(generatedResources);
+    }
+
+    @Override
+    public void addGeneratedResource(@NonNull String resource) {
+        info("Adding generated resource: " + resource, null);
```

Comment:
Why is this info statement here seems like debug output?

Reply:
I suppose it doesn't need to be. Do we have some guidelines for when to use `info` messages during TypeElement visitor processing?

Reply:
It's in the `JavaVisitorContext` too.

Reply:
For me this adds no value to the user. Output should be generated to point user to a generated file but not this kind of debut output IMO

Reply:
OK. I'll remove it from the java and Groovy visitor contexts.